Objects/Objects/BuiltElements/Beam.cs

```diff
@@ -141,10 +141,10 @@ public class ArchicadBeam : Beam
       get => archicadLevel;
       internal set
       {
-        if (value is not ArchicadLevel l)
+        if (value is ArchicadLevel or null)
```

Comment:
Can we fix the braces aroind the if, the throw doens't need to be in an else as it will throw if not entering the if

Reply:
@BovineOx @jozseflkiss
The throw Does need to be in an else here? otherwise it will always throw. We aren't returning here.